pages/resources/clan.mdx

```diff
@@ -0,0 +1,161 @@
+| badge_color_primary   | string           | The primary color of the badge as a hexadecimal color string                |
+| badge_color_secondary | string           | The secondary color of the badge as a hexadecimal color string              |
+| banner                | integer          | Number representing a [banner style](#banner-style)                         |
+| brand_color_primary   | string           | The primary color of the banner and some text as a hexadecimal color string |
+| brand_color_secondary | string           | The secondary color of the banner as a hexadecimal color string             |
+| wildcard_descriptors  | array[string]    | A list of 3 words to describe the clan                                      |
+| badge_hash            | string           | The clan's [badge hash](/reference#cdn-formatting)                          |
+| banner_hash           | string           | The clan's [banner hash](/reference#cdn-formatting)                         |
+
+###### Playstyle
+
+| Value | Name             | Title              | Subtitle                                                 | Emoji |
+| ----- | ---------------- | ------------------ | -------------------------------------------------------- | ----- |
+| 0     | NONE             |                    |                                                          |       |
+| 1     | SOCIAL           | "Very Casual"      | "Skill doesn't matter. We have a good time win or lose." | 👋     |
+| 2     | CASUAL           | "Casual"           | "We play to chill and have fun, but more fun if we win!" | 🎮     |
+| 3     | COMPETITIVE      | "Competitive"      | "We play together to win, rank up, or beat challenges."  | ⚔️     |
+| 4     | CREATIVE         |                    |                                                          |       |
+| 5     | VERY_COMPETITIVE | "Very Competitive" | "We won't sleep until we win."                           | 💀     |
+
+###### Badge Kind
+
+| Value | Name       |
+| ----- | ---------- |
+| 0     | SWORD      |
+| 1     | WATER_DROP |
+| 2     | SKULL      |
+| 3     | TOADSTOOL  |
+| 4     | MOON       |
+| 5     | LIGHTNING  |
+| 6     | LEAF       |
+| 7     | HEART      |
+| 8     | FIRE       |
+| 9     | COMPASS    |
+| 10    | CROSSHAIRS |
+| 11    | FLOWER     |
+| 12    | FORCE      |
+| 13    | GEM        |
+| 14    | LAVA       |
+| 15    | PSYCHIC    |
+| 16    | SMOKE      |
+| 17    | SNOW       |
+| 18    | SOUND      |
+| 19    | SUN        |
+| 20    | WIND       |
+
+###### Banner Style
+
+| Value | Name        |
+| ----- | ----------- |
+| 0     | NIGHT_SKY   |
+| 1     | CASTLE      |
+| 2     | WORLD_MAP   |
+| 3     | SEA_FOAM    |
+| 4     | WARP_TUNNEL |
+| 5     | HOUSE       |
+| 6     | HEIGHT_MAP  |
+| 7     | MESH        |
+| 8     | SPATTER     |
+
+## Endpoints
+
+<RouteHeader method="POST" url="/clan/{guild.id}">
+  Convert guild to clan
+</RouteHeader>
+
+Enables clan features for a guild
```

Comment:
```suggestion
Enables clan features for a guild. Returns a 204 empty response on success. Fires a [Guild Update](/topics/gateway-events#guild-update) Gateway event.
```
The response type is a guess based off the fact that the client ignores the response